src/components/Table/docs/Table.stories.mdx

```diff
@@ -0,0 +1,130 @@
+import { LiveTable } from './LiveTable';
```

Comment:
Shouldn't each component have at least one `Story`? I'm not very familiar with Storybook and I'm not sure what the canvas will show f we don't have any

Reply:
Yeah, you are correct, we should declare at least the most important test cases with `<Canvas>` and `<Story>` components. e.g. here we could have at very least 3 stories for each table row style. 

I'd also probably add story for table with active row, loading state, sortable table.

Reply:
There are interactive examples on the page, I'm not sure how we can make stories out of them because they have to contain internal state